ci-operator/config/openshift/cluster-ingress-operator/openshift-cluster-ingress-operator-master.yaml

```diff
@@ -197,6 +197,15 @@ tests:
         requests:
           cpu: 100m
     workflow: openshift-e2e-openstack-ipi
+- as: hypershift-e2e-aks
```

Comment:
We also have `e2e-hypershift`, which looks like the AWS version. Is there a reason the naming convention is different? Should the AWS job be called `hypershift-e2e-aws` to make it clear it's AWS now, or are these two fundamentally different jobs?